megamek/src/megamek/common/SpecialHexDisplay.java

```diff
@@ -23,14 +24,16 @@
 import megamek.common.util.ImageUtil;
 import megamek.common.util.fileUtils.MegaMekFile;
 
+import static megamek.client.ui.swing.tileset.TilesetManager.FILENAME_ORBITAL_BOMBARDMENT_INCOMING_IMAGE;
+
 /**
  * @author dirk
  */
 public class SpecialHexDisplay implements Serializable {
     private static final long serialVersionUID = 27470795993329492L;
 
     public enum Type {
-        ARTILLERY_AUTOHIT(new MegaMekFile(Configuration.hexesDir(), "artyauto.gif").toString()) {
+        ARTILLERY_AUTOHIT(new MegaMekFile(Configuration.hexesDir(), "artyauto.gif")) {
```

Comment:
so far, changing this from string to MegaMegFile seem to be working as expected. This also gave more flexibility on how to access those image files.